### PR TITLE
Use docker minikube driver

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

File: scripts/setup-minikube-for-linux.sh [hunk 1]
```diff
@@ -17,8 +17,8 @@ mkdir -p $HOME/.kube
 touch $HOME/.kube/config
 
 export KUBECONFIG=$HOME/.kube/config
-sudo -E minikube start --vm-driver=none
-sudo -E minikube addons enable ingress
+minikube start --driver=docker
```
Comment (patriknw) [inline scripts/setup-minikube-for-linux.sh:20]: do we have to update deployment guide and samples also?

Comment (chbatey) [reply scripts/setup-minikube-for-linux.sh:20]: i'll check. we shouldn't be recommending the any driver but maybe we do

Comment (chbatey) [reply scripts/setup-minikube-for-linux.sh:20]: We just assume users have minikube in the akka management one: https://doc.akka.io/docs/akka-management/current/kubernetes-deployment/deploying.html#deploying-to-minikube
and we link to the docs from the Lagom one

**Comment by raboof** [comment]
Should fix #746

**Comment by chbatey** [comment]
Merging this as all other PR validation is broken

**Pull request merged**